elfi/methods/post_processing.py

```diff
@@ -0,0 +1,233 @@
+"""
+Post-processing for posterior approximations from other ABC algorithms.
+
+See the review paper:
+
+Fundamentals and Recent Developments in Approximate Bayesian Computation
+Lintusaari et. al
+Syst Biol (2017) 66 (1): e66-e82.
+https://doi.org/10.1093/sysbio/syw077
+
+for more information.
+"""
+
+from sklearn.linear_model import LinearRegression
+import numpy as np
+
+from . import results
+
+
+__all__ = ('LinearAdjustment', 'adjust_posterior')
```

Comment:
The benefit is that one does not have to update this when new functions/classes are added. I don't see any benefit having this, since the functionality is exactly the same with or without this currently. But nevertheless, it's not that big of a deal :)

On the side note, you're probably right that it would be better to explicitly list the things in the `__init__` rather than use the star there. If you'd like to update that, it would be nice.